snap/snapcraft.yaml

```diff
@@ -19,6 +19,8 @@ architectures:
 environment:
   GITEA_CUSTOM: "$SNAP_COMMON"
   GITEA_WORK_DIR: "$SNAP_DATA"
```

Comment:
Are you able to test what changing this env var to `$SNAP_COMMON` would effect the running of Gitea? As right now when snaps upgrade the repositories are lost, and they should be stored in the common directory so they exist between upgrades.

Reply:
Are you referring to the issue described in https://github.com/go-gitea/gitea/issues/12292? In any case this should be tackled in a separate PR.